src/main/java/racinggame/domain/Car.java

```diff
@@ -0,0 +1,41 @@
+package racinggame.domain;
+
+import java.util.Objects;
+
+public class Car {
+
+    private static final int MOVE_BOUND = 4;
+    private int position;
```

Comment:
```suggestion
    private static final int MOVE_BOUND = 4;

    private int position;
```
상수와 멤버변수는 개행으로 구분하면 좋을 것 같아요